jni/src/main/java/com/horizen/vrfnative/VRFOutput.java

```diff
@@ -0,0 +1,27 @@
+package com.horizen.vrfnative;
+
+import com.horizen.librustsidechains.FieldElement;
+import com.horizen.librustsidechains.Library;
+
+
+public class VRFOutput {
```

Comment:
VRFOutput can be confused with the real output of the VRF. Let's call it VRFProveResult that is containing the vfrProof and the vrfOutput